lms/templates/message.html.jinja2

```diff
@@ -0,0 +1,9 @@
+{% extends 'templates/base.html.jinja2' %}
```

Comment:
Do we really want to extend `base.html.jinja2` here?

If we just want the unsubscribed page to be as plain and fast-loading is possible then I think `base.html.jinja2` may have some stuff that we don't want: a Google site verification code (I can't remember what this is for), a CSS stylesheet, favicons.

I also worry about changes to this base template for the purposes of whatever other pages use the base template causing these unsubscribe pages to break.

We might want a separate `base.plain.html.jinja2` for the unsubscribe templates to extend?

```jinja2
# base.plain.html.jinja2
<!doctype html>
<html lang=en>
  <head>
    <meta charset=utf-8>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>{{ title }}</title>
  </head>
  <body>
    <main>
      <h1>{{ title }}</h1>

      {% block content %}{% endblock %}
    </main>
  </body>
</html>
```